com.github.wwmm.easyeffects.yml

```diff
@@ -35,7 +35,7 @@ add-extensions:
     directory: extensions/Plugins
     version: "20.08"
     add-ld-path: lib
-    merge-dirs: ladspa;lv2
+    merge-dirs: ladspa;lv2;LSP;ZamPlugins
```

Comment:
that's not how you do it.

Reply:
Here is an example on how to automatically install audio plugin extensions 

https://github.com/flathub/org.nongnu.gsequencer.gsequencer/blob/master/org.nongnu.gsequencer.gsequencer.json#L39

Reply:
Thanks, that makes more sense, I think it should be built in properly now.

I still need to test how well the integration works.

Reply:
Ok, so there is [confirmation](https://github.com/wwmm/easyeffects/issues/1068#issuecomment-892356420) that Flatpak plugins work here, thanks for the advice.

I am wondering why with these builds the plugin extensions aren't auto-installed? When I install gsequencer, flatpak installs `org.freedesktop.LinuxAudio.Plugins.CMT`, but with these test builds that doesn't happen for `LSP` or `ZamPlugins`.

As far as I can tell the manifests do the same thing. Perhaps Flathub test builds work differently than stable ones?

Reply:
if you `flatpak update` it will likely try to pull them

Reply:
Flatpak update doesn't seem to try to pull them here.
`flatpak update -v` doesn't even output anything referring to them.

Reply:
maybe it's because they are on different remotes :-/

Reply:
Yeah, I am not sure how to workaround that then. I guess there isn't a need to since it should work once it's on Flathub proper.